deploy/helm/sumologic/values.yaml

```
@@ -4061,7 +4065,19 @@ metadata:
               - batch
             exporters:
               - sumologic/systemd
-
+          logs/otc/containers:
```

```suggestion
          logs/otlp/containers:
```

I'd rename this as the important bit here is actually the underlying protocol being used - `otlp` - not the agent.